examples/jsm/controls/ArcballControls.js

```diff
@@ -2874,7 +2875,7 @@ class ArcballControls extends EventDispatcher {
 			this._v3_1.set( this._v2_1.x, this._v2_1.y, - 1 );
 			this._v3_1.applyMatrix4( camera.projectionMatrixInverse );
 
-			const rayDir = this._v3_1.clone().normalize(); //unprojected ray direction
+			const rayDir = this._v3_2.copy( _v3_1 ).normalize(); //unprojected ray direction
```

Comment:
The controls are now broken because `_v3_1` is `undefined`. 

Have you tested the controls with this change? Notice the runtime error only appears by interacting with `misc_controls_arcball`. Just loading the example is not sufficient.

Reply:
Sorry, I didn't... (But tested now) Thank you!